src/OpenTelemetry.Contrib.Extensions.PersistentStorage/FileStorage.cs

```diff
@@ -20,7 +20,7 @@
 using System.Linq;
 using System.Timers;
 
-namespace OpenTelemetry.Contrib.PersistentStorage
+namespace OpenTelemetry.Contrib.Extensions.PersistentStorage
```

Comment:
Minor - would you consider the C# 10.0 way of saving 4 spaces? 😄

https://github.com/open-telemetry/opentelemetry-dotnet/blob/1150f4a4e43608800b5571da499ed53f237c3904/test/OpenTelemetry.Tests.Stress/Skeleton.cs#L26

Reply:
Done

Reply:
ran into an issue with this. (CSC : error CS1617: Invalid option '10.0' for /langversion. Use '/langversion:?' to list supported values.) . Will investigate and update this in separate PR.